.github/ISSUE_TEMPLATE/bug-report.yaml

```
@@ -0,0 +1,44 @@
+name: Bug report
+description: The mod doesn't work? Use this form to open a new bug report
+title: "[Bug]: <title>"
+type: bug
+body:
+  - type: markdown
+    attributes:
+      value: Please fill out this form to help us solve your issue
+
```

```suggestion
  - type: markdown
    attributes:
      value: |
        > [!NOTE]
        > For tech support questions please open [a new discussion](https://github.com/Monika-After-Story/MonikaModDev/discussions/categories/qa-tech-support-general)
```

And maybe some callout in case it's not issue worthy?